### PR TITLE
Replace PlotMessage with TextComponents

### PR DESCRIPTION
**Pull request by MattBDev**
Replaces PlotMessage with TextComponents.

### CODE REVIEW

File: Bukkit/src/main/java/com/github/intellectualsites/plotsquared/bukkit/listeners/PlayerEvents.java [hunk 1]
```diff
@@ -268,53 +278,6 @@ public static boolean checkEntity(Entity entity, Plot plot) {
 
     @EventHandler public void onRedstoneEvent(BlockRedstoneEvent event) {
         Block block = event.getBlock();
-/*        switch (block.getType()) {
```
Comment (boy0001) [inline Bukkit/src/main/java/com/github/intellectualsites/plotsquared/bukkit/listeners/PlayerEvents.java]: Unrelated to this PR, but why was this commented out?

File: Bukkit/src/main/java/com/github/intellectualsites/plotsquared/bukkit/listeners/PlayerEvents.java [hunk 2]
```diff
@@ -60,8 +67,10 @@
 /**
  * Player Events involving plots.
  */
-@SuppressWarnings("unused") public class PlayerEvents extends PlotListener implements Listener {
+@SuppressWarnings("unused")
+public class PlayerEvents extends PlotListener implements Listener {
 
+    Color
```
Comment (N0tMyFaultOG) [inline Bukkit/src/main/java/com/github/intellectualsites/plotsquared/bukkit/listeners/PlayerEvents.java]: gg

File: Bukkit/src/main/java/com/github/intellectualsites/plotsquared/bukkit/listeners/PlayerEvents.java (hunk 1 above)
Comment (MattBDev) [reply Bukkit/src/main/java/com/github/intellectualsites/plotsquared/bukkit/listeners/PlayerEvents.java]: I'm not sure why. I didn't mean to commit this code in this PR.

File: Bukkit/src/main/java/com/github/intellectualsites/plotsquared/bukkit/listeners/PlayerEvents.java (hunk 2 above)
Comment (MattBDev) [reply Bukkit/src/main/java/com/github/intellectualsites/plotsquared/bukkit/listeners/PlayerEvents.java]: Oops.

**Comment by MattBDev** [comment]
Closing this PR since it isn't quite ready yet.

**Pull request closed without merging**